src/Installer.php

```diff
@@ -59,6 +59,16 @@ class Installer
 HELP;
 
     /**
+     * The api url to determine the available versions of puli
+     */
+    const VERSION_API_URL = '%s://puli.io/versions.json';
```

Comment:
this should be changed to `%s://puli.io/download/versions.json` as I changed the location of the file.